modules/nf-core/snpeff/build.sh

```diff
@@ -18,9 +18,11 @@ build_push() {
     docker push nfcore/snpeff:${SNPEFF_VERSION}.${GENOME}
 }
 
+build_push "CanFam3.1" "99"  "5.1"
 build_push "GRCh37"    "87"  "5.1"
 build_push "GRCh38"    "105" "5.1"
```

Comment:
you wanna update the humans while you are at it or later?

Reply:
later